packages/components/psammead-section-label/src/index.test.jsx

```diff
@@ -47,7 +47,64 @@ describe('SectionLabel', () => {
           script={arabic}
           dir="rtl"
           labelId="test-section-label"
+          service="persian"
+        >
+          بعض محتوى النص
+        </SectionLabel>,
+      );
+    });
+
+    describe('With linking title', () => {
+      shouldMatchSnapshot(
+        'should render correctly',
+        <SectionLabel
+          script={latin}
+          labelId="test-section-label"
           service="news"
+          href="/igbo/other-index"
+          linkText="See All"
+        >
+          This is text in a linking SectionLabel.
+        </SectionLabel>,
+      );
+
+      shouldMatchSnapshot(
+        'should render correctly with explicitly showing the bar',
```

Comment:
Just curious what effect explicitly showing the bar has, since the bar is shown by default ?